src/toggle/Toggle.tsx

```diff
@@ -0,0 +1,73 @@
+import "./_toggle.scss";
+
+import React, {createContext, useState} from "react";
+import classNames from "classnames";
+
+import ToggleItem from "./item/ToggleItem";
+
+export interface ToggleItemShape {
+  id: string;
+  children: React.ReactNode;
+}
+
+const ToggleContext = createContext({
+  selectedToggleItemsState: [] as ToggleItemShape[],
+  setSelectedToggleItemsState: (() => null) as React.Dispatch<
+    React.SetStateAction<ToggleItemShape[]>
+  >,
+  isMultiple: false,
+  onToggleItem: (id: string) => id
+});
+
+export interface ToggleProps {
+  children: React.ReactNode;
+  onToggle: (id: string) => void;
+  isMultiple?: boolean;
+  position?: "vertical" | "horizontal";
+  isDisabled?: boolean;
+  customClassName?: string;
+  testid?: string;
+}
+
+function Toggle({
+  children,
+  onToggle,
+  isMultiple = false,
+  position = "horizontal",
+  isDisabled,
+  customClassName,
+  testid
+}: ToggleProps) {
+  const toggleClassName = classNames("toggle", customClassName, {
+    "toggle--is-horizontal": position === "horizontal",
+    "toggle--is-vertical": position === "vertical",
+    "toggle--is-disabled": isDisabled
+  });
+  const [selectedToggleItemsState, setSelectedToggleItemsState] = useState<
+    ToggleItemShape[]
+  >([]);
+
+  return (
+    <div data-testid={testid} className={toggleClassName}>
```

Comment:
I believe we should wrap Toggle component body with a `List` instead of div, because it is actually a list with `li` items (`ToggleItem`)

Reply:
`ToggleItem` is a `ListItem` variant actually but `Toggle` provides a context. We don't know how many items should exist in `ToggleContext` and we can't pass `items` prop that `ListItem's`. I'm not sure we can use `List`.

Reply:
Yeah I didn't think of that. Then, I think we can directly use `ul` here (instead of `div`)
Because at any case, the children will be `li`s

Reply:
Absolutely right! Thank you 🚀